### PR TITLE
feat: adding toolbar as a new window

### PR DESCRIPTION
**Pull request by RohanPunjani**

### CODE REVIEW

**Comment by clearlysid** [comment]
haha 🤣 this is why cross-platform testing for small features is important. I'll push a fix for this on macOS soon.

https://github.com/clearlysid/helmer-micro/assets/30227512/c49037d2-47ae-4a4d-81ff-5d89a4e6e75b

File: src-tauri/src/toolbar/mod.rs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+#[tauri::command]
+pub async fn show_toolbar(button_coords: Vec<u32>, area: Vec<u32>, app: AppHandle) {
+    println!("show_toolbar here!");
+    if app.get_window("toolbar").is_none() {
+        crate::toolbar::init_toolbar(&app);
+    }
+    println!("Coordinates {:?}", button_coords);
+    let toolbar_win = app.get_window("toolbar").unwrap();
+    let pos = Position::Physical((button_coords[0], button_coords[1]).into());
```
Comment (clearlysid) [inline src-tauri/src/toolbar/mod.rs:50]: > let pos = Position::Physical((button_coords[0], button_coords[1]).into());

Changing this to `Position::Logical` fixes the mismatching positioning issue for macOS. Can you try that and check if Windows works fine with it too? (i think it should)

File: src/pages/cropper.astro [hunk 2]
```diff
@@ -73,7 +68,7 @@ import Base from "../components/base.astro";
 		display: block;
 	}
 
-	button {
+	/* button {
```
Comment (clearlysid) [inline src/pages/cropper.astro:71]: I think we can just remove these parts. Don't foresee us coming back to these styles.

**Review by clearlysid — CHANGES_REQUESTED** [review]
Looks good overall, left a few comments. Can merge to main once they're resolved.

File: src-tauri/src/toolbar/mod.rs (hunk 1 above)
Comment (RohanPunjani) [reply src-tauri/src/toolbar/mod.rs:50]: Yep, that works on windows too 🙌🙌

File: src/pages/cropper.astro (hunk 2 above)
Comment (RohanPunjani) [reply src/pages/cropper.astro:71]: Okay

**Pull request merged**